models/iaf_cond_alpha_mc_fixedca.cpp

```diff
@@ -0,0 +1,853 @@
+  //!< Waveform for calcium spike current
```

Comment:
Please also leave a reference to the paper, to document where the magic numbers are coming from.